platforms/pf_test/include/platform.hpp

```diff
@@ -34,6 +34,7 @@
 #include "odometry.hpp"
```

Comment:
Who could use that ?

Reply:
Not sure which commit you are referring to...

Probably `expose uartpb on public interface` ?
If yes, I dropped that commit because it will concern the next PR about uartpb refactoring.